### PR TITLE
RET-5243

### PR DESCRIPTION
**Pull request by cindychmcts**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/RET-5243

### Change description ###

When I'm in the 'Reply to a referral' event, I want to be able to open the DCF

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by lvo700 — APPROVED** [review]

**Review by fergusoshea — APPROVED** [review]

File: definitions/json/AuthorisationCaseField/AuthorisationCaseField.json [hunk 1]
```diff
@@ -15197,6 +15197,90 @@
     "UserRole": "et-acas-api",
     "CRUD": "R"
   },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "replyToReferralDcfLink",
+    "UserRole": "caseworker-employment-legalrep-solicitor",
+    "CRUD": "D"
+  },
```
Comment (hsjhita) [inline definitions/json/AuthorisationCaseField/AuthorisationCaseField.json:15205]: can we remove this please? legal rep shouldn't have access

File: definitions/json/AuthorisationCaseField/AuthorisationCaseField.json [hunk 2]
```diff
@@ -15197,6 +15197,90 @@
...
+    "CaseFieldID": "replyToReferralDcfLink",
+    "UserRole": "et-acas-api",
+    "CRUD": "R"
+  },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "replyToReferralDcfLinkLabel",
+    "UserRole": "caseworker-employment-legalrep-solicitor",
+    "CRUD": "D"
+  },
```
Comment (hsjhita) [inline definitions/json/AuthorisationCaseField/AuthorisationCaseField.json:15247]: can we remove this please? legal rep shouldn't have access

File: definitions/json/AuthorisationCaseField/AuthorisationCaseField.json (hunk 1 above)
Comment (hsjhita) [reply definitions/json/AuthorisationCaseField/AuthorisationCaseField.json:15205]: ignore me, just seen it's D

File: definitions/json/AuthorisationCaseField/AuthorisationCaseField.json (hunk 2 above)
Comment (hsjhita) [reply definitions/json/AuthorisationCaseField/AuthorisationCaseField.json:15247]: ignore me, just seen it's D

**Review by hsjhita — APPROVED** [review]

**Pull request merged**